include/tvm/runtime/packed_func.h

```diff
@@ -559,7 +569,18 @@ class TVMMovableArgValue_ : public TVMPODValue_ {
 
  private:
   /*! \return The arg value repr of the value. */
-  TVMArgValue AsArgValue() const { return TVMArgValue(value_, type_code_); }
+  TVMArgValue AsArgValue() const { return TVMArgValue(value_, type_code_, prefix_message_); }
+  /*! \brief Construct a message to provide context to any conversion errors in TVMPODValue_. */
+  static std::string error_string(const std::string& name, int argnum) {
```

Comment:
Please do not write a static function in c++ header...Use inline to make sure only one copy of this function exists after linking.